index.html

```diff
@@ -26,37 +26,321 @@ <h2 class="bottom_color">I'm a software developer </h2>
+                    </div>
+                        <div class="action1">
+                            <button type="button" class="button"><p class="btx1">See Project</p></button>
+                        </div>
+                        
+                         </div>
+                         
+                           
+
+                        
+                    </div>
+
+                </div>
+                <div class="card">
+                    <div class="image_placeholder_card">
+                       
+                    <div class="text">
+                       
+                        <div>
+                        <p class="project_title">Multi-Post Stories Gain+Glory</p>
+                    </div>
+                    <div class="tags">
+                        <ul class="tag">
+                            <li class="tag1">Ruby on rails</li>
+                            <li class="tag2">css</li>
+                            <li class="tag3">JavScript </li>
+                            <li class="tag4">html</li>
+                        </ul>                                                  
+                    </div>
+                        <div class="action1">
+                            <button type="button" class="button"><p class="btx1">See Project</p></button>
+                        </div>
+                        
+                         </div>
+                         
+                           
+
+                        
+                    </div>
+
+                </div>
+                <div class="card">
+                    <div class="image_placeholder_card">
+                       
+                    <div class="text">
+                       
+                        <div>
+                        <p class="project_title">Multi-Post Stories Gain+Glory</p>
+                    </div>
+                    <div class="tags">
+                        <ul class="tag">
+                            <li class="tag1">Ruby on rails</li>
+                            <li class="tag2">css</li>
+                            <li class="tag3">JavScript </li>
+                            <li class="tag4">html</li>
+                        </ul>                                                  
+                    </div>
+                        <div class="action1">
+                            <button type="button" class="button"><p class="btx1">See Project</p></button>
+                        </div>
+                        
+                         </div>
+                         
+                           
+
+                        
+                    </div>
+
+                </div>
+                <div class="card">
+                    <div class="image_placeholder_card">
+                       
+                    <div class="text">
+                       
+                        <div>
+                        <p class="project_title">Multi-Post Stories Gain+Glory</p>
```

Comment:
- [ ] Kindly I recommend to you use  a header tag `hx` the project title instead of `p` tag ,This way will be semantically correct and the screen readers and other technologies will thank you. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info.